molo/core/tasks.py

```diff
@@ -72,77 +72,99 @@ def promote_articles():
         featured_in_homepage=True)
 
 
+def get_days(section=None, site_settings=None):
```

Comment:
I think you should split this out: `def get_days_from_section(section)` and `def get_days_from_settings(settings)`